docs/kyma/13-04-common-errors.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Common installation errors
+type: Troubleshooting
+---
```

Comment:
For unsuccessful installation, I would also describe how to find out what has failed.

It is also worth to execute `helm history {RELEASE} --tls -o yaml` because it shows errors that occured on each revision. Sometimes you have for example an error where you have a typo in the charts, it will be reported but with next set of retries the origin error will be "covered" with a different error which is a consequence of the first one like `{RELEASE} already exsits`. Thanks to helm history we can see the root error

Reply:
Added this one to the `Common Errors` doc 👍